plugins/deps/CMakeLists.txt

```diff
@@ -13,37 +13,20 @@ find_package(Git REQUIRED)
 # =============================================================================
 # libsonata
 # =============================================================================
-if(BRAYNS_CIRCUITEXPLORER_ENABLED OR BRAYNS_DTI_ENABLED AND NOT sonata_FOUND AND NOT TARGET sonata::sonata_shared)
+if(BRAYNS_CIRCUITEXPLORER_ENABLED AND NOT sonata_FOUND AND NOT TARGET sonata::sonata_shared)
     if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/libsonata")
         message(STATUS "Downloading libsonata from submodules")
         FetchContent_Declare(
             libsonata
             GIT_REPOSITORY https://github.com/BlueBrain/libsonata.git
-            GIT_TAG v0.1.21
+            GIT_TAG v0.1.22
             GIT_SHALLOW ON
             GIT_SUBMODULES_RECURSE ON
             SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/libsonata
         )
         FetchContent_Populate(libsonata)
     endif()
 
-    # MorphIO requires latest version of HighFive. (2.7.1)
```

Comment:
bah, next time I'll try to warn you, to save you this; you would have caught the mismatch - sorry

Reply:
No worries, it was a good CMake exercise anyway :)